templates/credentials/sign-up.html

```diff
@@ -1,24 +1,28 @@
 {% extends "credentials/base_cred.html" %}
 
-{% block title %}Sign Up for Beta | Certification{% endblock %}
+{% block title %}Sign Up for Beta | Credentials{% endblock %}
+
+{% block body_class %}
+  is-paper
```

Comment:
Why not do it on `base_cred.html` level? Now some pages are white and some are gray.